client/fe/webpack/optimization.js

```diff
@@ -18,7 +18,7 @@ const chunkSplitting = {
     core: {
       name: 'core',
       chunks: 'all',
-      minChunks: 40,
+      minChunks: 10, // Lowered from 40 - extract code used in 10+ pages instead of 40+
```

Comment:
I don't think that comment is very useful as after the merge only a git history dive will explain where the 40 came from but it also doesn't hurt :)